id-repository/id-repository-identity-service/Dockerfile

```diff
@@ -97,6 +97,7 @@ EXPOSE 8090
 ENTRYPOINT [ "./configure_start.sh" ]
 
 CMD wget -q --show-progress "${kernel_ref_idobjectvalidator_url}" -O "${loader_path_env}"/kernel-ref-idobjectvalidator.jar; \
+    wget "${iam_adapter_url_env}" -O "${loader_path_env}"/kernel-auth-adapter.jar; \
```

Comment:
why are we doing this change ? I believe we more towards taking in the auth adapter as compile time dependency.